src/windows/com.salesforce.plugin.smartstoreProxy.js

```diff
@@ -24,7 +24,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-    var SALESFORCE_MOBILE_SDK_VERSION = "4.0.0";
+    var SALESFORCE_MOBILE_SDK_VERSION = "4.2.0";
```

Comment:
Doesn't matter. `unstable` has the upcoming version number, not the current prod one.